src/matrix.ts

```diff
@@ -63,9 +63,23 @@ export * as SecretStorage from "./secret-storage";
 export type { ICryptoCallbacks } from "./crypto"; // used to be located here
 export { createNewMatrixCall } from "./webrtc/call";
 export type { MatrixCall } from "./webrtc/call";
-export { GroupCallEvent, GroupCallIntent, GroupCallState, GroupCallType } from "./webrtc/groupCall";
+export {
+    GroupCallEvent,
+    GroupCallIntent,
+    GroupCallState,
+    GroupCallType,
+    GroupCallStatsReportEvent,
+} from "./webrtc/groupCall";
 export type { GroupCall } from "./webrtc/groupCall";
 export { CryptoEvent } from "./crypto";
+export { SlidingSyncEvent } from "./sliding-sync";
+export { VerificationRequestEvent, VerifierEvent } from "./crypto-api/verification";
```

Comment:
these are already exposed via `export * as Crypto` below (and are documented as such in the jsdoc). Please don't clutter the main namespace more than it has to be.

Reply:
The "rule", at least in my head, is that crypto-related stuff should be contained within the `Crypto` namespace. `CryptoEvent` predates my recent work in this area so hasn't (yet) been moved. (And if it was moved, we'd still have to export it at the top level for backwards compat.)